### PR TITLE
Add LABY.net

### PR DESCRIPTION
**Pull request by Northernside**
LABY.net is a web-based lookup tool for Minecraft accounts & capes; LabyMod cloaks & statistics; and even more!

### CODE REVIEW

File: websites/L/LABY.net/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
+{
+    "$schema": "https://schemas.premid.app/metadata/1.3",
+    "service": "LABY.net",
+    "author": {
+      "name": "Northernside",
+      "id": "434417514332815370"
+    },
+    "category": "games",
+    "logo": "https://labymod.net/page/tpl/assets/images/white_wolf.png",
+    "thumbnail": "https://cdn.discordapp.com/attachments/260475731182616576/862003257693175808/labynet_announcement.png",
```
Comment (Bas950) [inline websites/L/LABY.net/dist/metadata.json:10]: upload to imgur

File: websites/L/LABY.net/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+      "stats",
+      "cloaks",
+      "capes",
+      "game",
+      "gaming",
+      "lookup"
+    ],
+    "url": "laby.net",
+    "description": {
+      "en": "LABY.net is a web-based lookup tool for Minecraft accounts & capes; LabyMod cloaks & statistics; and even more!"
```
Comment (Bas950) [inline websites/L/LABY.net/dist/metadata.json:30]:
```suggestion
      "en": "A modern & fast Minecraft profile search. Find skins, capes, servers, statistics and more on LABY.net"
```

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/L/LABY.net/dist/metadata.json (hunk 1 above)
Comment (Northernside) [reply websites/L/LABY.net/dist/metadata.json]: Done

**Comment by Bas950** [comment]
Upload proof of the presence working

**Comment by Northernside** [comment]
> Upload proof of the presence working

Should I send screenshots of every presence state or just one?

**Comment by Bas950** [comment]
> > Upload proof of the presence working
> 
> Should I send screenshots of every presence state or just one?

At least 2 or 3

**Comment by Northernside** [comment]
> > > Upload proof of the presence working
> > 
> > 
> > Should I send screenshots of every presence state or just one?
> 
> At least 2 or 3

https://imgur.com/a/AwH91E7 there are 3 screenshots uploaded

**Review by Bas950 — DISMISSED** [review]

File: websites/L/LABY.net/presence.ts [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    presenceData.details = `Viewing server ${item}`;
+  } else if(document.location.pathname.startsWith("/badge")) {
+    const item = document.querySelector("div.mb-1>h1").textContent;
+    presenceData.details = `Viewing badge ${item}`;
+  } else if(document.location.pathname.startsWith("/cape")) {
+    const item = document.querySelector("div.mb-1>h1").textContent;
+    presenceData.details = `Viewing cape ${item}`;
+  }
+
+  if(presenceData.details === null) {
```
Comment (SlowLife1661) [inline websites/L/LABY.net/presence.ts:37]:
```suggestion
  if(!presenceData.details) {
```

**Review by Slowlife01 — CHANGES_REQUESTED** [review]

File: websites/L/LABY.net/presence.ts [hunk 4]
```diff
@@ -0,0 +1,43 @@
+const presence = new Presence({
+    clientId: "867452106016161822"
+  }),
+
+  browsingStamp = Math.floor(Date.now() / 1000);
+  // I just did what Codacy told me even though it looks weird lol
```
Comment (Bas950) [inline websites/L/LABY.net/presence.ts:6]:
```suggestion
  }),
  browsingStamp = Math.floor(Date.now() / 1000);
```
Just ignore it if it errors about it again.

**Review by Bas950 — APPROVED** [review]

**Review by Slowlife01 — APPROVED** [review]

**Pull request merged**